src/buffer/buffer.rs

```diff
@@ -186,65 +181,65 @@ impl Buffer {
     }
 
     /// Print a string, starting at the position (x, y)
-    ///
-    /// `style` accepts any type that is convertible to [`Style`] (e.g. [`Style`], [`Color`], or
-    /// your own type that implements [`Into<Style>`]).
```

Comment:
The removed comment is pretty standard language we use across the library. It's focused on newer users that might not be as savvy about how the `Into` idiom works as a prompt to find out more. While definitely not necessary from a more advanced user perspective, I'd prefer to keep this.

Reply:
Users are unlikely playing around with the buffer directly. Only people implementing widgets are at the buffer.

Also: I dislike Into because it's less clear what happens. This only adds to it. Requiring a comment a dozen times all over the place to clarify something hard to understand hints a code smell that should be improved.

Reply:
I understand your concern over the use of `Into`, but it's something we're definitely keeping. Requiring code to construct a full Style in order to set colors or modifiers is overly verbose.

> Only people implementing widgets

I've generally been pushing the idea that implementing widgets is a thing that almost every developer should do in most apps (`impl Widget for &App`). E.g. see https://github.com/ratatui-org/ratatui/pull/1079. This is in contrast to writing functions that call methods on Frame, which tends to produce more convoluted apps.

Ideally in the future, this method should not generally be used. People should just use Span / Line / Text instead of this for greater flexibility.

Reply:
Resolving for now as I don't feel strongly enough about this specific use to block it.